src/record/record-transition.js

```diff
@@ -67,6 +68,31 @@ RecordTransition.prototype.hasVersion = function( version ) {
 };
 
 /**
+ * Send version exists error if the record has been already loaded, else
+ * store the version exists error to send to the sockerWrapper once the 
+ * record is loaded
+ *
+ * @param   {SocketWrapper} socketWrapper the sender
+ * @param   {Number} version The version number
+ *
+ * @public
+ */
+RecordTransition.prototype.sendVersionExists = function( socketWrapper, version ) {
```

Comment:
I'm a bit confused: How do we know the record's version before it has been loaded?